doc/source/cluster/kubernetes/user-guides/observability.md

```diff
@@ -0,0 +1,78 @@
+(kuberay-observability)=
+
+# KubeRay Observability
+
+## Ray Dashboard
+
+* To view the Ray dashboard running on the head Pod, users can follow [these instructions](kuberay-port-forward-dashboard).
+* The Ray dashboard also integrates with Prometheus and Grafana. See [Using Prometheus and Grafana](kuberay-prometheus-grafana) for more details.
+* The Ray dashboard also supports the "CPU Flame Graph" and "Stack Trace" features. Refer to [Profiling with py-spy](kuberay-pyspy-integration) to enable these features.
+
+## KubeRay Observability
+
+### Method 1: Check KubeRay operator's logs for errors
+
+```bash
+kubectl logs $KUBERAY_OPERATOR_POD -n $YOUR_NAMESPACE | tee operator-log
+```
+
+The command redirects the operator's logs to a file called `operator-log`. You can then search for errors in the file.
+
+### Method 2: Check custom resource status
+
+```bash
+kubectl describe [raycluster|rayjob|rayservice] $CUSTOM_RESOURCE_NAME -n $YOUR_NAMESPACE
+```
+
+Check the status and events of the custom resource to see if there are any errors.
+
+### Method 3: Check logs of Ray Pods
+
+Check the Ray logs directly by accessing the log files on the Pods. See [Ray Logging](configure-logging) for more details.
+
+```bash
+kubectl exec -it $RAY_POD -n $YOUR_NAMESPACE -- bash
+# Check the logs under /tmp/ray/session_latest/logs/
+```
+
+(kuberay-port-forward-dashboard)=
+### Method 4: Check Dashboard
+
+```bash
+export HEAD_POD=$(kubectl get pods --selector=ray.io/node-type=head -o custom-columns=POD:metadata.name --no-headers)
+kubectl port-forward $RAY_POD -n $YOUR_NAMESPACE --address 0.0.0.0 8265:8265
+# Check $YOUR_IP:8265 in your browser
```

Comment:
Does `localhost:8265` always work? If so it might be better, so users don't have to look up how to find `YOUR_IP`

Reply:
> Does localhost:8265 always work?

No. If users run KubeRay in their EC2 devbox, they cannot use `localhost:8265` directly (some tricks can make this work, but this is a bit beyond the scope of this discussion).

Reply:
Update https://github.com/ray-project/ray/pull/39074/commits/7c39cd2d2e8b9f983995c573188a0582a94ceab9